airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -283,7 +283,7 @@ def _get_ti_pod_labels(context: Optional[dict] = None, include_try_number: bool
             return {}
 
         ti = context['ti']
-        run_id = getattr(ti, 'run_id') or context['run_id']
+        run_id = getattr(ti, 'run_id', context['run_id'])
```

Comment:
Maybe simply `context["run_id"]` since that key is always present anyway?

Reply:
Doh!

Some tests will need correcting as they call this work a minimal context rather than a real one.